day2/kanban/lib/kanban/data/project.ex

```diff
@@ -3,20 +3,39 @@ defmodule Kanban.Data.Project do
 
   import Ecto.Changeset
 
-  # alias Kanban.Data.{Task}
+  alias Kanban.Data.{Task, Issue, Project}
+
+  @type embeds_one(t) :: [t]
```

Comment:
Мне очень понравилось желание описать тип, но не нужно это делать в сотне разных мест :) Типы из соседних модулей прекрасно доступны.
```suggestion
  @type embeds_one(t) :: t
```